packages/patternlab/styleguide/source/_patterns/05-pages/binder-page-internal.json

```diff
@@ -213,35 +213,40 @@
           "link": {
             "href": "#",
             "info": "",
-            "text": "Rules of Criminal Procedure"
+            "text": "Rules of Criminal Procedure",
+            "srLabel": "This page is part of Rules of Criminal Procedure category."
           }
         },
         "prev": {
           "arrowButton": {
             "href": "#",
-            "info": "",
-            "direction": "left"
+            "info": "Previous page, Rule 99: A Rule About Something",
+            "direction": "left",
+            "hidden": true
```

Comment:
can we name this var `hidden` --> `ariaHidden` instead? I feel like it's clearer what it's going to do.

Reply:
Changed as suggested.